README.md

```diff
@@ -1,4 +1,28 @@
 # Kotlin Wallet SDK
 
 Kotlin Wallet SDK is a library that allows developers to build wallet applications on the Stellar network faster. It
-utilizes [Java Stellar SDK](https://github.com/stellar/java-stellar-sdk) to communicate with a Stellar Horizon network.
\ No newline at end of file
+utilizes [Java Stellar SDK](https://github.com/stellar/java-stellar-sdk) to communicate with a Stellar Horizon network.
+
+## Wallet
+
+`class Wallet(horizonUrl: String, horizonPassphrase: String)`
+
+Example
+
+```kotlin
+// Using Horizon testnet configuration
+val wallet = Wallet("https://horizon-testnet.stellar.org", "Test SDF Network ; September 2015")
+```
+
+### `Wallet.create()`
+
+`fun create(): AccountKeypair(val publicKey: String, val secretKey: String)`
+
+Example
+
+```kotlin
+val newAccountKeypair = wallet.create()
+
+// publicKey=GB7S6OXH4UBA4QLR2OR4IXAOW7KOG67SUBEIXHGMJDNRKADHW7QGETPQ,
```

Comment:
Although this is commented out and the keys are test keys, some code analyzer will complain about these keys. It is better to read these from external configurations.

Reply:
These are examples of generated keypairs. I'll use shortened version instead for security.